src/pages/demo/partials/staffProfiles.tsx

```diff
@@ -0,0 +1,60 @@
+import { drupalClient } from '@/utils/drupalClient'
+import {
+  StaffProfiles,
+  StaffNewsProfiles,
+} from '@/components/paragraph/staff_profile'
+import { DrupalJsonApiParams } from 'drupal-jsonapi-params'
+import Container from '@/components/container'
+
+const StaffProfilePage = ({ nodes }) => {
+  if (!nodes) return
+  return (
+    <>
+      <Container className="container">
+        <div id="content" className="interior">
+          <main className="va-l-detail-page va-facility-page">
+            <div className="usa-grid usa-grid-full">
+              <div className="usa-width-three-fourths">
+                {nodes.map((node, index) => (
+                  <div key={index}>
+                    <StaffProfiles node={node} />
+                    <StaffNewsProfiles node={node} />
```

Comment:
Added news profile to this section for demo purposes, we need to think about how we want to showcase the components we build